editor/src/clj/editor/render_target.clj

```diff
@@ -49,7 +49,7 @@
                           :label "format"
                           :type :choicebox
                           :options (protobuf-forms/make-options (protobuf/enum-values Graphics$TextureImage$TextureFormat))
-                          :default (ffirst (protobuf/enum-values Graphics$TextureImage$TextureFormat))}]}
+                          :default Graphics$TextureImage$TextureFormat/TEXTURE_FORMAT_RGBA}]}
```

Comment:
Does this work? In `material.clj` we specify the default as the keyword value, i.e. `:semantic-type-none` for `Graphics$VertexAttribute$SemanticType`.